src/main/java/frc/robot/subsystems/arm/commands/SetElbowAngle.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.arm.ArmConstants;
+
+public class SetElbowAngle extends CommandBase {
+    private final Arm arm;
+    private final double angle;
+
+    public SetElbowAngle(double angle) {
+        this.arm = Arm.getInstance();
+        this.angle = angle;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void initialize() {
+        arm.setElbowJointAngle(angle);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return MathUtil.applyDeadband(angle - arm.getElbowJointAngle(), ArmConstants.SETPOINT_DEADBAND) == 0;
```

Comment:
1. This is not the same as deadband, "tolerance" is a better name (talking about the constant).
2. You don't need `MathUtil.applyDeadband()` here, you can just check whether `abs(angle - arm.getElbowJointAngle()) < Constants.SETPOINT_TOLERANCE`.
3. You repeat this logic between commands. Extract it into a method in the `Arm` class.
4. You should check for the velocity, too, and not just for the position. If the elbow passes through its setpoint but at a high speed, it will overshoot.